applications/FSIapplication/fsi_application.cpp

```diff
@@ -28,6 +28,7 @@ KRATOS_CREATE_VARIABLE(double, MAPPER_SCALAR_PROJECTION_RHS);
 KRATOS_CREATE_VARIABLE(double, SCALAR_PROJECTED);
 KRATOS_CREATE_VARIABLE(double, FICTITIOUS_FLUID_DENSITY);
 KRATOS_CREATE_VARIABLE(double, FSI_INTERFACE_RESIDUAL_NORM);
+KRATOS_CREATE_VARIABLE(double, FSI_INTERFACE_MESH_RESIDUAL_NORM);
```

Comment:
Is created, defined, and registered in python but not in kratos.

Maybe missing:
```C++
KRATOS_REGISTER_VARIABLE(FSI_INTERFACE_MESH_RESIDUAL_NORM);
```